### PR TITLE
update : add reference to issue number

### LINKED ISSUE #188: Update Hacktoberfest template for Issue reference
PR's should reference the issue number for ease of review and closure.

### PR DESCRIPTION
**Pull request by SubhamChoudhury**
This fixes #188 

# Language

What language was used for the solution?
- None
# Challenge

What is the name of the challenge being completed?
- Add reference to issue number for ease of review and closure.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/hacktoberfest-template.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ assignees: ''
 # Participation
 
 All Hacktoberfest solutions should be provably correct. That is, solution submissions should either include the unit tests from the original challenge (implemented in your language of choice) or at least print out the expected answers when run by the maintainers. Solutions that do not meet these criteria may be rejected, or flagged as 'invalid' or 'spam' in egregious cases. This can result in your PR not counting towards your Hacktoberfest participation or becoming ineligible to participate in Hacktoberfest. Violations of our [Code of Conduct](https://github.com/codeconnector/CodingDojo/blob/main/CODE_OF_CONDUCT.md) will be reported, and will also render you ineligible to participate in Hacktoberfest. Please be respectful of the maintainers' and other participants' time by doing your best to submit real solutions to the puzzles. Thanks, and welcome to our learning community!
```
Comment (xanderyzwich) [inline .github/ISSUE_TEMPLATE/hacktoberfest-template.md:11]: Please add a line of whitespace before the new section

**Review by xanderyzwich — CHANGES_REQUESTED** [review]

**Comment by SubhamChoudhury** [comment]
Please let me know if any other changes are required!

**Review by ericwburden — APPROVED** [review]
This looks good to me. I modified the title format slightly to be more in line with the titles we've been using on issues of this type.

**Review by xanderyzwich — APPROVED** [review]
This looks good

**Review by ericwburden — APPROVED** [review]

**Pull request merged**